assets/templates/header-component.html

```diff
@@ -167,6 +167,23 @@
                           comprar
                         </button>
                       </div>
+                      <div class="user-plan--type">
+                        <p><strong>498</strong> Suscriptores disponibles</p>
+                        <div class="dp-request-sent">
+                          <button
```

Comment:
If changes are added into the html take in accound that they should be added into webapp too. 
To validate what I'm asserting please navigate into new reports to check how this dropdown is showing.

Reply:
just add a new item and we check it against previous versions. They are all in the library.
Being a new element, it should not impact what is already there.
I tried the library side in all the cases we use and it works fine
@nreal will have to test how it looks on the screens you mention.

- https://cdn.fromdoppler.com/doppler-ui-library/build1241/nav-user-component.html
- https://cdn.fromdoppler.com/doppler-ui-library/build1241/header-component.html#